decidim/decidim
# Include only public entities in the following page

#### :tophat: What? Why?
This PR fixes #8178, filtering the entities shown in the Follows page. It checks for visibility information for participatory spaces and resources.

This change will affect the page performance, so the queries were optimized to be as fast as possible, but we will need to check the performance on big instances of Decidim.

#### :pushpin: Related Issues
- Fixes #8178

#### Testing
1. Sign in as administrator
2. Create a new assembly. Do NOT publish it.
3. As a visitor, go to the administrator public profile
4. Go to the "Follows" tab (http://HOST/profiles/USER/following)
5. Check that the unpublished assembly doesn't appear in the page.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
